test_scripts/API/VehicleData/WindowStatus/015_Resumption_subscription_to_windowStatus_param_IGN_OFF_ON_cycle.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0261-New-vehicle-data-WindowStatus.md
+-- Description: Check that SDL resumes the subscription for 'windowStatus' after IGN_OFF/ON cycle
+-- In case:
+-- 1) App is subscribed to `windowStatus` data.
+-- 2) IGN_OFF/IGN_ON cycle is performed.
+-- 3) App re-registered with actual HashId.
+-- SDL does:
+--  a) send VehicleInfo.SubscribeVehicleData(windowStatus=true) request to HMI.
+-- 4) HMI sends VehicleInfo.SubscribeVehicleData response to SDL.
+-- SDL does:
+--  a) not send SubscribeVehicleData response to mobile app
+-- 5) HMI sends valid OnVehicleData notification with all parameters of `windowStatus` structure.
+-- SDL does:
+--  a) process this notification and transfer it to mobile app.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/WindowStatus/common')
+
+--[[ Local Function ]]
+local function subscribeVD(pRPC, pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      common.setHashId(data.payload.hashID, pAppId)
+    end)
+  common.subUnScribeVD(pRPC, pAppId)
+end
+
+local function checkResumption_FULL()
```

Comment:
this function and for NONE level are used in several scripts so can be moved to common file